src/js/components/DataTable/stories/Placeholder.js

```diff
@@ -23,15 +23,11 @@ export const Placeholder = () => (
             gap="small"
             background={{ color: 'background-front', opacity: 'strong' }}
           >
-            <Box
-              direction="row"
+            <Spinner
```

Comment:
Can we just use the default Spinner, rather than customizing the border, since this story isn't really about the Spinner?

Reply:
Sure, I just kept it as you originally defined it. I'll clean it up.